tox.ini

```diff
@@ -8,30 +9,41 @@ envlist =
 
 
 [testenv]
-# This avoids rebuilding and installing all rust dependencies
-# which is especially useful for local testing.
-skip_install = True
-# TODO: Allow for using usedevelop by adjusting setup.py
-# usedevelop = True
-passenv =
-    RASCALINE_BUILD_TYPE
-    RUSTUP_HOME
-    CARGO*
+package = external
+package_env = build-rascaline
+passenv = *
 
+rascaline-dependencies =
+    equistore-core @ https://github.com/lab-cosmo/equistore/archive/3bced4c.zip\#subdirectory=python/equistore-core
```

Comment:
Remind me why do we need this here? Shouldn't this be come automatically from the package dependencies?

Reply:
because we (i.e. tox, with the `package=env` option) install with `--no-deps`

Reply:
ahh okay, maybe lets add a comment line there that I am will not ask again the next time we touch this file.